src/app/api/template/route.test.ts

```diff
@@ -0,0 +1,114 @@
+import { MessageType, StatusCodeType } from '@/lib/response/responseUtil';
+import { DELETE } from './route';
+import { prisma, clearDatabase } from '@/lib/prisma';
+import { NextRequest } from 'next/server';
+import { createMocks } from 'node-mocks-http';
+import { Role } from '@prisma/client';
+import { getServerAuthSession } from '@/lib/auth';
+
+const TrainerUser1 = {
```

Comment:
variables TrainerUser1 and TrainerUser2 don't follow camelCase convention

Reply:
Fixed